pkg/kube/apis/extendedjob/v1alpha1/types.go

```diff
@@ -3,7 +3,7 @@ package v1alpha1
 import (
 	"fmt"
 
-	corev1 "k8s.io/api/core/v1"
+	bacthv1 "k8s.io/api/batch/v1beta1"
```

Comment:
The import alias has a typo. bacthv1 -> batchv1.

Reply:
thx